### PR TITLE
Use latest operator version. Face-lift of WebServer example.

### PR DESCRIPTION
**Pull request by csviri**

### CODE REVIEW

File: webserver/k8s/crd.yaml [hunk 1]
```diff
@@ -1,23 +1,35 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```
Comment (metacosm) [inline webserver/k8s/crd.yaml:1]: This should be generated automatically using the CRD generator.

**Review by metacosm — APPROVED** [review]
LGTM but I haven't tested the changes.

File: webserver/k8s/crd.yaml (hunk 1 above)
Comment (attilameszaros-form3) [reply webserver/k8s/crd.yaml:1]: I tested it for websierver at least, today doing a presentation on CNCF meetup, it looked ugly that we still use the old beta api.

Well not sure we want ot have it in every example, users's not necessary want to generate yaml  I guess. But we can make at least one example that way in future.

Comment (csviri) [reply webserver/k8s/crd.yaml:1]: I tested it for websierver at least, today doing a presentation on CNCF meetup, it looked ugly that we still use the old beta api.

Well not sure we want ot have it in every example, users's not necessary want to generate yaml I guess. But we can make at least one example that way in future.

**Pull request merged**